solana/pyth2wormhole/client/src/attestation_cfg.rs

```diff
@@ -134,9 +144,10 @@ pub struct AttestationConditions {
     #[serde(default = "default_max_batch_jobs")]
     pub max_batch_jobs: usize,
 
-    /// Trigger attestation if price changes by the specified percentage.
+    /// Trigger attestation if price changes by the specified
+    /// percentage, expressed in integer parts per thousand.
     #[serde(default)]
-    pub price_changed_pct: Option<f64>,
+    pub price_changed_ppt: Option<u64>,
```

Comment:
i really really think you should use basis points here. 1 / 1000th is a weird unit for prices, but 1 / 10000 (1 bp) is very standard. This configuration option is going to confuse people.